Added examples of License uses for microbiological data (#158)
NFDI4Microbiota/nfdi4microbiota-knowledge-base: docs/_Research-Data-Management/26-licenses.md

## Usage
Creators can declare a works license by selecting it during upload to a website/database/repository or attaching a line stating the title, author and license to the published work.

Here we look at some examples of license usage, found on [Zenodo](https://zenodo.org/).

> ### [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) example usage

Reviewer: HAving hyperlinks in headings is usually frowned upon as it can collide with autogenerated permalink/anchors for the headings in TOC or cause accessibility issues in some browsers
```suggestion
### CC BY 4.0 example usage
```

We are going to be looking at two examples here: 
1. [Metagenome-assembled genomes (MAGs), colorectal cancer (CRC)](https://zenodo.org/records/7008911) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7008911.svg)](https://doi.org/10.5281/zenodo.7008911)
2. [Metagenome assemblies and metagenome-assembled genomes from the Daphnia magna microbiota](https://zenodo.org/records/4435010) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4435010.svg)](https://doi.org/10.5281/zenodo.4435010)

> Both datasets have been submitted to Zenodo under the CC BY 4.0 license. That means that the data that has been deposited is free to share and free to redistribute (including commercially) in any format or medium. It also enables the data user to build upon or transform it for any purpose (including commercial use). However, the license requires the data user to give appropriate credit to the submitter/data generator. In addition, the reuser must also provide a link to the license and disclose any changes made when licensing their work when derived from work already under deed license.

Reviewer: ```suggestion
Both datasets have been submitted to Zenodo under the [CC BY 4.0 license](https://creativecommons.org/licenses/by/4.0/) . That means that the data that has been deposited is free to share and free to redistribute (including commercially) in any format or medium. It also enables the data user to build upon or transform it for any purpose (including commercial use). However, the license requires the data user to give appropriate credit to the submitter/data generator. In addition, the reuser must also provide a link to the license and disclose any changes made when licensing their work when derived from work already under deed license.
```

I'm not sure reuser is a term, care to switch to user?

Author: ```suggestion
Both datasets have been submitted to Zenodo under the [CC BY 4.0 license](https://creativecommons.org/licenses/by/4.0/). That means that the data that has been deposited is free to share and free to redistribute (including commercially) in any format or medium. It also enables the data user to build upon or transform it for any purpose (including commercial use). However, the license requires the data user to give appropriate credit to the submitter/data generator. In addition, the user must also provide a link to the license and disclose any changes made when licensing their work when derived from work already under license.
```

> ### [CC0](https://creativecommons.org/publicdomain/zero/1.0/) example usage

Reviewer: ```suggestion
### CC0 example usage
```

same comment as above regarding the headings

We are going to be looking at two examples here:
1. [Data from: Mining for NRPS and PKS genes revealed a high diversity in the Sphagnum bog metagenome](https://zenodo.org/records/4976456) [![DOI](https://zenodo.org/badge/DOI/10.5061/dryad.hf56v.svg)](https://doi.org/10.5061/dryad.hf56v)
2. [Data from: Generational conservation of composition and diversity of field-acquired midgut microbiota in Anopheles gambiae sensu lato during colonization in the laboratory](https://zenodo.org/records/5001400) [![DOI](https://zenodo.org/badge/DOI/10.5061/dryad.98jj7gk.svg)](https://doi.org/10.5061/dryad.98jj7gk)

> Both datasets have been submitted to Zenodo under the CC0 license. That means that the data that has been deposited is in the public domain. That implies that the data deposited can be copied, modified, distributed, and used even for commercial purposes, and the depositor/generator of the data waives their right to the work. The reuser of data does not need to seek the permission of the data/material submitter or generator.

Reviewer: ```suggestion
Both datasets have been submitted to Zenodo under the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/) . That means that the data that has been deposited is in the public domain. That implies that the data deposited can be copied, modified, distributed, and used even for commercial purposes, and the depositor/generator of the data waives their right to the work. The reuser of data does not need to seek the permission of the data/material submitter or generator.
```

Author: ```suggestion
Both datasets have been submitted to Zenodo under the  [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/). That means that the data that has been deposited is in the public domain. That implies that the data deposited can be copied, modified, distributed, and used even for commercial purposes, and the depositor/generator of the data waives their right to the work. The user of data does not need to seek the permission of the data/material submitter or generator.
```
